src/gui/update.rs

```diff
@@ -414,13 +414,8 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
                             {
                                 a.release_channel = *release_channel;
                             } else {
-                                // Else we try to determine the release_channel based of installed version.
```

Comment:
I think we just delete the else block here since `ReleaseChannel::Default` is the `Default` and what gets set when we create an addon.